applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -106,6 +109,42 @@ static int next_free_audio_stream_index_get(uint8_t *index)
 	return -ENOSPC;
 }
 
+/**
+ * @brief  Get the presentation delay for al headsets
+ *
+ * @param  sel_max Selection of the use of the minimum or maximum
+ *                 preferred presentation delay (TRUE = max; FALSE = min)
+ * @param  index   The index of the hesdset to test against
+ * @param  pd      Pointer to the absolute presentation delay for the
+ *                 set of headsets
+ * @param  pref_pd Pointer to the preferred presentation delay for the
+ *                 set of headsets
+ */
+static void find_heaset_pres_delay(bool sel_max, int index, uint32_t *pd, uint32_t *pref_pd)
```

Comment:
```suggestion
static void find_headset_pres_delay(bool sel_max, int index, uint32_t *pd, uint32_t *pref_pd)
```

Reply:
would it be cleaner to have separate functions for this?
`find_headset_pres_delay_max` and `find_headset_pres_delay_min`?